package.json

```diff
@@ -33,6 +33,7 @@
     "lint:js": "./node_modules/.bin/eslint \"src/**/*.js\"",
     "lint:styles": "stylelint \"src/**/*.scss\"",
     "lint": "npm run lint:html && npm run lint:styles && npm run lint:js",
+    "fix": "./node_modules/.bin/eslint --fix \"src/**/*.js\" && stylelint --fix \"src/**/*.scss\"",
```

Comment:
You can write your own scripts in `npm`. See how the script above is called `npm run fix`? This is where you tell the fix command what to do! It's the exact same thing as if you ran what's in the quotes here right in terminal.